include/fi.h

```diff
@@ -185,6 +191,20 @@ static inline int ofi_equals_sockaddr(struct sockaddr_in *addr1,
                 (addr1->sin_port == addr2->sin_port));
 }
 
+static inline int ofi_tr_addr_format(int family)
```

Comment:
tr is short for translate (after the linux tr command). If it is not obvious, I can rename it to ofi_translate_addr_format.